packages/core/src/config/load.ts

```diff
@@ -6,7 +6,7 @@ import { Config, DOMAINS } from './config';
 import { transformConfig } from './utils';
 import { resolveConfig } from './config-resolvers';
 
-import type { RawConfig, Region } from './types';
+import type { DeprecatedInRawConfig, RawConfig, Region } from './types';
```

Comment:
`DeprecatedInRawConfig` or `DeprecatedRawConfig`?

Reply:
I reckon it should be `In` as it's only about fields being deprecated inside the raw config and not the entire deprecated raw config.